apprentice-react-o-shop/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
```

Comment:
these image files clutter up the repo, let's get rid of the files and the references to them